docs/sources/operators-guide/reference-http-api/index.md

```diff
@@ -52,6 +52,7 @@ This document groups API endpoints by service. Note that the API endpoints are e
 | [Label values cardinality](#label-values-cardinality)                                 | Querier, Query-frontend        | `GET, POST <prometheus-http-prefix>/api/v1/cardinality/label_values`      |
 | [Build information](#build-information)                                               | Querier, Query-frontend, Ruler | `GET <prometheus-http-prefix>/api/v1/status/buildinfo`                    |
 | [Get tenant ingestion stats](#get-tenant-ingestion-stats)                             | Querier                        | `GET /api/v1/user_stats`                                                  |
+| [Query-scheduler ring status](#query-scheduler-ring-status)                           | Query-scheduler                | `GET /query-scheduler/ring`                                               |
```

Comment:
Why do query-frontends and queriers not expose this ring?

Reply:
This is a good feedback. As we discussed in a previous call, let me open an issue and then try to address it in a consistent way in Mimir (e.g. store-gateway ring is currently not exposed by queriers, and doc about `/ingester/ring` is missing to mention it's also exposed by distributors, queriers, rulers).

Reply:
See: https://github.com/grafana/mimir/issues/2977